### PR TITLE
[Azure Data Factory] Adding refresh and get IntegrationRuntimeObjectMetadata API

### PR DESCRIPTION
**Pull request by meiyan-Li**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by KrisBash — APPROVED** [review]
LGTM for ARM

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json [hunk 1]
```diff
@@ -4558,6 +4658,16 @@
...
         }
       }
+    },
+    "GetSsisObjectMetadataRequest": {
+      "type": "object",
+      "description": "The request payload of get ssis object metadata.",
+      "properties": {
+          "metadataPath": {
+              "type": "string",
+              "description": "Metadata path"
```
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json]: Needs dot at the end

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json [hunk 2]
```diff
@@ -1269,6 +1269,106 @@
...
+        "tags": [
+          "integrationRuntimeObjectMetadata"
+        ],
+        "operationId": "IntegrationRuntimeObjectMetadata_Refresh",
+        "x-ms-examples": {
+          "IntegrationRuntimeObjectMetadata_Refresh": {
+            "$ref": "./examples/IntegrationRuntimeObjectMetadata_Refresh.json"
+          }
+        },
+        "description": "Refresh a ssis integration runtime object metadata.",
```
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json]: Change to "SSIS"

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json [hunk 3]
```diff
@@ -1269,6 +1269,106 @@
...
+        "tags": [
+          "integrationRuntimeObjectMetadata"
+        ],
+        "operationId": "IntegrationRuntimeObjectMetadata_Get",
+        "x-ms-examples": {
+          "IntegrationRuntimeObjectMetadata_Get": {
+            "$ref": "./examples/IntegrationRuntimeObjectMetadata_Get.json"
+          }
+        },
+        "description": "Get a ssis integration runtime object metadata by specified path. The return is pageable metadata list.",
```
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json]: Same here, and any other if there are more

File: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/examples/IntegrationRuntimeObjectMetadata_Refresh.json [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    "resourceGroupName": "exampleResourceGroup",
+    "factoryName": "exampleFactoryName",
+    "integrationRuntimeName": "testactivityv2",
+    "api-version": "2018-06-01"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Date": "Mon, 03 Dec 2018 06:24:38 GMT",
+        "Location": "https://management.azure.com/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.DataFactory/factories/exampleFactoryName/integrationRuntimes/testactivityv2/refreshObjectMetadata/operationresults/ca63c855b72d44959653ffcc6eb0b96c?api-version=2018-06-01",
```
Comment (hvermis) [inline specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/examples/IntegrationRuntimeObjectMetadata_Refresh.json]: This URI does not conform to ARM standard, will email you separately

Comment (hvermis) [reply specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/examples/IntegrationRuntimeObjectMetadata_Refresh.json]: It doesn't block your swagger changes

**Review by hvermis — APPROVED** [review]

**Comment by meiyan-Li** [comment]
@sergey-shandar Could you help merge the code?

**Comment by sergey-shandar** [comment]
@meiyan-Li could you fix model validation errors?

**Comment by amarzavery** [comment]
@meiyan-Li There are model validation failures. Please fix them. You can take a look at the logs over [here](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/464773467#L574).

**Review by amarzavery — CHANGES_REQUESTED** [review]
fix model validation failures.

**Comment by meiyan-Li** [comment]
@amarzavery I have fixed my model validation issue. For the existing operation model validation issue, hermins have a backlog to fix it. Please see this PR for more info: https://github.com/Azure/azure-rest-api-specs/pull/3957

**Comment by meiyan-Li** [comment]
@amarzavery Please help merge this PR

**Review by amarzavery — APPROVED** [review]

**Pull request merged**